org-formation/070-guard-duty/_tasks.yaml

```diff
@@ -0,0 +1,75 @@
+Parameters:
+  <<: !Include '../_parameters.yaml'
+
+  appName:
+    Type: String
+    Default: 'guardduty'
+
+  accountId:
+    Type: String
+    Description: The identifier from the account used to manage GuardDuty
+    Default: !Ref SecurityCentralAccount
+
+GuardDuty:
+  Type: update-stacks
+  Template: https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/master/templates/GuardDuty/guard-duty.yaml
+  StackName: !Sub '${resourcePrefix}-${appName}'
+  StackDescription: GuardDuty - Base
+  TerminationProtection: true
+  DefaultOrganizationBindingRegion: !Ref primaryRegion
+  DefaultOrganizationBinding:
+    Account: !Ref accountId
+  OrganizationBindings:
+    LogArchiveBinding:
+      Account: !Ref LogArchiveAccount
+    MemberBinding:
+      Account: '*'
+      IncludeMasterAccount: true
+      ExcludeAccount:
+        - !Ref accountId
+    AllBinding:
+      Account: '*'
+      IncludeMasterAccount: true
+  Parameters:
+    resourcePrefix: !Ref resourcePrefix
+    accountId: !Ref accountId
+
+# bucket that contains the trusted IP addresses that are excluded from GuardDuty findings
```

Comment:
I suggest not adding any trusted ips list

Reply:
i think it's nice to have it as an option.  I mean if you don't put anything on the list then it's basically the same as not having it there at all.